### PR TITLE
feat: Hero section change

### LINKED ISSUE #8684: Hero section change
let's make landing prettier on the desktop as well. 

image

same layout on mobile - icon for the network, divider with text 

updated in figma: https://www.figma.com/file/JXNtryaNYhY0XJNArzvPQx/Handoff?type=design&node-id=6301-28493&mode=design&t=V3vseTYynUTC6EZi-11

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

  👇 __ Let's make a quick check before the contribution.

  ## PR Type

  - [x] Feature

  ## Needs Design check

  - @exezbcz please review

  ## Context

  - [x] Closes #8684

  #### Did your issue had any of the "$" label on it?

  - [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)

  ## Screenshot 📸

  - [x] My fix has changed UI

image

  ## Copilot Summary
  copilot:summary

  copilot:poem

### CODE REVIEW

**Comment by Jarsen136** [comment]
The e2e test failure would be fixed by #8687

**Review by preschian — APPROVED** [review]

File: components/landing/HeroBanner.vue [hunk 1]
```diff
@@ -16,6 +31,20 @@
...
+          {{ $t('search.landingTitle1') }}
+          <span
+            class="subtitle is-size-1 is-size-2-mobile has-text-weight-bold has-text-centered capitalize ml-4">
+            {{ $t('search.landingTitle2') }}
+          </span>
+        </h1>
+        <h1
+          class="is-size-1 is-size-2-mobile has-text-weight-bold has-text-centered">
+          {{ $t('search.landingTitle3') }}
+        </h1>
```
Comment (roiLeo) [inline components/landing/HeroBanner.vue:46]: multiple `h1` 

**suggestion:**: wrap text in `span` instead

File: components/landing/HeroBanner.vue [hunk 2]
```diff
@@ -37,14 +66,37 @@
...
     </div>
+
+    <template v-if="!isMobile">
+      <img
+        src="/landing-blurred-header-right.webp"
+        class="landing-search-right"
+        width="740"
+        height="1000"
+        alt="" />
+      <img :src="landingImage[1]" class="landing-shapes" alt="" />
```
Comment (roiLeo) [inline components/landing/HeroBanner.vue:80]: missing alt

File: components/landing/HeroBanner.vue [hunk 3]
```diff
@@ -1,11 +1,26 @@
...
       src="/landing-mobile-shape.svg"
       alt="landing page background" />
+    <template v-else>
+      <img
+        src="/landing-blurred-header-left.webp"
+        class="landing-search-left"
+        width="740"
+        height="1000"
+        alt="" />
+      <img :src="landingImage[0]" class="landing-shapes" alt="" />
```
Comment (roiLeo) [inline components/landing/HeroBanner.vue:19]: missing alt

**Review by roiLeo — APPROVED** [review]
otherwise lgtm

**Comment by exezbcz** [comment]
just to be sure, on kodadot.xyz, only 3 networks will be shown right?

**Comment by Jarsen136** [comment]
> just to be sure, on kodadot.xyz, only 3 networks will be shown right?

YES.

File: components/landing/HeroBanner.vue (hunk 1 above)
Comment (Jarsen136) [reply components/landing/HeroBanner.vue]: updated

File: components/landing/HeroBanner.vue (hunk 3 above)
Comment (Jarsen136) [reply components/landing/HeroBanner.vue]: added

**Comment by yangwao** [comment]
Thanks!
pay 50 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 6.99 USD/DOT ~ 7.153 $DOT
🧗 [16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z](https://polkadot.subscan.io/account/16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)
🔗 [0x4d9e2d4e2d6da40784bb4c709a137808d8b20c0e3017ab3793b75aa8f414f121](https://polkadot.subscan.io/extrinsic/0x4d9e2d4e2d6da40784bb4c709a137808d8b20c0e3017ab3793b75aa8f414f121)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**